amplify.yml

```diff
@@ -6,6 +6,8 @@ backend:
         - nvm install 20.13.1
         - nvm use 20.13.1
         - npm ci --cache .npm --prefer-offline
+        - (cd lambdas/authorizer && npm ci)
```

Comment:
Minor, but do we need to run `npm ci` for these or can we just delete the lambdas folder like we do for the frontend?
I presume this will go away with 7005 anyway?

Reply:
yep all of these will go away with 7005

Reply:
but removed anyway - it was an alternative to your `rm -rf` solution that i put in before those changes were in this branch